<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBOXDialog</class>
 <widget class="QDialog" name="BBOXDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="chooseLayerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Layer Extent:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="chooseBBOXLayer">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>475</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Layer Extent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>500</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply BBOX</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomIn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zoomOut">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="panButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
